package/gluon-mesh-vpn-core/luasrc/usr/bin/gluon-wan

```diff
@@ -21,12 +21,13 @@ if ok ~= 0 then
 	io.stderr:write(string.format("gluon-wan: unable to change to group: %s\n", err))
 	os.exit(1)
 end
+local args = arg
```

Comment:
This assignment just hides from luacheck that `arg` is modified, but doesn't actually fix it.

I propose something like this to copy the table:
```lua
local args = {[0] = arg[1], unpack(arg)}
table.remove(args, 1)
```
```

Reply:
Modified as suggested